flowcraft/generator/components/assembly.py

```diff
@@ -4,6 +4,50 @@
 except ImportError:
     from flowcraft.generator.process import Process
 
+class Bcalm(Process):
+    """Bcalm process template interface
+
+    This process is set with:
+
+        - ``input_type``: fastq
+        - ``output_type``: assembly
+        - ``ptype``: assembly
+
+    """
+
+    def __init__(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        self.input_type = "fastq"
+        self.output_type = "fasta"
+
+        self.dependencies = ["integrity_coverage"]
+
+        self.params = {
+            "bcalmKmerSize": {
+                "default": 31,
+                "description":
+                    "size of a kmer"
+            },
+            "clearInput": {
```

Comment:
This parameter is not used in the nextflow process to remove temporary input files. If you really want to include it, you'll need to include the code that removes temporary input files (that is, input files that are genereated as part of the pipeline execution and stored somewhere in the `work` dir). You have an example of that here: https://github.com/assemblerflow/flowcraft/blob/master/flowcraft/generator/templates/pilon.nf#L31-L39
If not, could you remove this option?